assets/src/conversion/index.js

```diff
@@ -94,23 +134,40 @@ class Conversion extends Component {
 							</b>
 						</li>
 					</ul>
+
+					<br />
+
+					<h3>{ __( 'Reset Ongoing Conversion to Start Again' ) }</h3>
 					<p>
-						<i>
-							*{' '}
-							{ __(
-								'note: keep an eye on your active browser tabs currently performing the conversion'
-							) }
-						</i>
+						{ __(
+							'In case that your active conversion browser tab has been closed by accident, or it has been interrupted and closed unexpectedly, you may reset the conversion status here, and start converting all over again. Note that this will enable you to restart the conversion, but any previous results may be lost.'
+						) }
 					</p>
+					<input
+						type="submit"
```

Comment:
Maybe it's a matter of personal preference, but I'd replace these inputs with `button` elements, since that's what they are - there's no `form` to submit here ;)

Reply:
I'd take this advice here in normal cases, but Thomas will do a complete redesign of the UI very soon, so let's leave that up to him :)